contracts/src/common/IRegistryDatastore.sol

```diff
@@ -10,6 +10,7 @@ pragma solidity >=0.8.13;
  */
 interface IRegistryDatastore {
     event SubregistryUpdate(address indexed registry, uint256 indexed id, address subregistry, uint64 expiry, uint32 data);
+    // The expiry parameter is kept for backward compatibility but is always 0
```

Comment:
No need to keep expiry for backward compatibility as we haven't deployed in production yet. Please remove it from the event as well